Hw1/task2.py

```diff
@@ -0,0 +1,15 @@
+sides_of_a_triangle = input('Введите через запятую стороны треугольника (например 8,9,10 : ')
+a,b,c = sides_of_a_triangle.split(',')
+enter = str
+a = int(a)
+b = int(b)
+c = int(c)
```

Comment:
Можно сделать значительно проще.
```suggestion
sides_of_a_triangle = input('Введите через запятую стороны треугольника (например 8,9,10 : ')
a, b, c = map(int, sides_of_a_triangle.split(','))
```
Хотя лучше писать числа через пробел. В этом случае не нужно будет ничего передавать методу `split`.